docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,277 @@
+
+* Authors: Bruno Khélifi, Thomas Vuillaume
+* Created: May 25th, 2022
+* Accepted: Sept. 21th, 2022
+* Status: accepted
+
+Abstract
+========
+
+Given that the Gammapy library is more widely used by the community, a proper citation of the project including
+a policy about the authorship is necessary. This PIG addresses this issue by setting an authorship policy for the
+Gammapy project for each type of product.
+
+.. contents:: Table of Contents
+   :depth: 2
+
+1. Introduction
+===============
+
+Gammapy started in 2013 and is now widely used in scientific publications. A proper citation scheme with correct authorship allows:
+
+- a proper citation of the used Gammapy release,
+- a proper recognition of the achieved work of any contributor,
+- compliance with the FAIR principles for Research Software (`FAIR4RS <https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs>`_).
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should properly cite Gammapy.
+It is asked to cite the DOI of the used Gammapy version as well as the associated paper (e.g. the latest LTS
+release).
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ SWHID identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This citation scheme with these **two references** will be given on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors that requires to be updated with time according to a
+general policy. This section defines what is a `contributor` to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
```

Comment:
Contributions to Gammapy can be made ...

or

Contributing to Gammapy is possible ...